src/Symfony/Component/DependencyInjection/CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ CHANGELOG
  * Add `#[AsAlias]` attribute to tell under which alias a service should be registered or to use the implemented interface if no parameter is given
  * Allow to trim XML service parameters value by using `trim="true"` attribute
  * Allow extending the `Autowire` attribute
+ * Include value type when unable to dump a service container
```

Comment:
should be removed as we're going to merge this as "minor" (aka "not a feature nor a bug")